### PR TITLE
test build-n-pub with gh-actions

### PR DESCRIPTION
**Pull request by larsclaussen**

### CODE REVIEW

**Comment by larsclaussen** [comment]
I tested this from within a branch on `push` which produced a valid package (https://pypi.org/project/threedi-api-client/3.0.10.dev0/). For master I changed two things:

```
    tags:
      - '*' # Push events to matching v*, i.e. v1.0, v20.15.10
```

Additionally I added the build dependency:
```
jobs:
  publish:
    needs: build
```
and un-commented this line
```
      if: startsWith(github.ref, 'refs/tags')
```
So be when merged be careful with the next release - check carefully if everything went well!

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+name: Build and publish Python 🐍 distributions 📦 to PyPI and TestPyPI
```
Comment (jpprins1) [inline .github/workflows/release.yml:1]: Haha awesome!

**Review by jpprins1 — APPROVED** [review]
Looks good to me!

**Comment by larsclaussen** [comment]
@RichardB9 once this is merged no need to push to PyPI through fullrelease anymore!

**Pull request merged**